README.md

```diff
@@ -1,19 +1,41 @@
-# Deep Root
-> The soil moisture tracker to help track plant health. Final @makersacademy project by Team Cactus
-
 [![Build Status](https://travis-ci.com/breycarr/deep_root.svg?branch=master)](https://travis-ci.com/breycarr/deep_root) [![Maintainability](https://api.codeclimate.com/v1/badges/3dc1d964235155d65b53/maintainability)](https://codeclimate.com/github/breycarr/deep_root/maintainability) [![Test Coverage](https://api.codeclimate.com/v1/badges/3dc1d964235155d65b53/test_coverage)](https://codeclimate.com/github/breycarr/deep_root/test_coverage) [![BCH compliance](https://bettercodehub.com/edge/badge/breycarr/deep_root?branch=master)](https://bettercodehub.com/)
+
+# Project DEEPROOT by Team C A C T U S
+
+
+- MVP 1: An LED light display to track plant health.
+- MVP 2: A touchscreen to track plant health.
+
 ## Installation
-Ensure you have [Python3 or later](https://www.python.org/downloads/) installed, then clone this repository.
 
-Run `$ pip3 install -r requirements.txt` to install the necessary dependencies
+In your terminal, run:
 
-## Usage example
+```
+> python --version                           # check you have Python installed.
+> brew install python3            # run only if you don't have Python installed.
+> git clone <repo name>
+> pip3 install -r requirements.txt       # install the necessary dependencies
+```
 
-A few motivating and useful examples of how your product can be used. Spice this up with code blocks and potentially more screenshots.
+## Testing
 
-## Development setup
+This programme uses PyTest.
 
-Describe how to install all development dependencies and how to run an automated test-suite of some kind. Potentially do this for multiple platforms.
+```
+> python3 -m pytest
+```
+
+## Documentation
+
+We have a Project C A C T U S [blog](https://medium.com/@makers_c_a_c_t_u_s) on Medium, documenting significant moments, such as setting up the hardware for the first time or a fixing a particularly stubborn error.
+
+We used [Trello](https://trello.com/b/DZAhiebz/c-a-c-t-u-s) to divide our daily tasks and assign tickets to pairs. Checkout this repo's [wiki](https://github.com/breycarr/deep_root/wiki) for our group's daily diary using agile methodologies. You can also find the project's:
```

Comment:
Is our trello public?

Reply:
Will check now and set it if not, good spot